api/.swagger2-codegen-ignore

```diff
@@ -22,18 +22,22 @@
 # Then explicitly reverse the ignore rule for a single file:
 #!docs/README.md
 
-README.md
-pom.xml
-.gitignore
-.travis.yml
-build.sbt
-docs/*.md
-settings.gradle
-git_push.sh
-gradle.properties
-src/main/AndroidManifest.xml
-src/main/resources/application.properties
-src/test/**
-gradlew
-gradlew.bat
-gradle/**
+build/swagger2/README.md
```

Comment:
nit: you could just do `**/*.md` in case you want another top-level markdown file.

Reply:
This is an ignore file for the output of Swagger codegen, it doesn't relate to the files that we check in to the repository. I also don't see a reason to make changes in this PR since only the base path changed.